test/azure/legacy/Expected/AcceptanceTests/PackageModeBatch/azure/packagemode/batch/_version.py

```diff
@@ -0,0 +1,8 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+VERSION = "1.0.0"
```

Comment:
should be
`from .paging._version import VERSION`

Reply:
updated